pkg/engine/utils.go

```diff
@@ -235,7 +235,7 @@ func MatchesResourceDescription(resourceRef unstructured.Unstructured, ruleRef k
 	}
 
 	// creating final error
-	var errorMessage = "rule not matched:"
+	var errorMessage = fmt.Sprintf("rule %s not matched:", ruleRef.Name)
```

Comment:
What does "rule <%s> was not matched" mean? 

Is this indicating that the admission request / resource did not match the rule, or that the rule resulted in an a policy failure (i.e. violation).

I think its the latter, in which case the message is misleading.

The message could be, "rule %s result = fail" or "rule %s failed".

Reply:
It's actually the former, this function `MatchesResourceDescription` evaluates if the given resource matches the rule.

Reply:
Ok, then it seems fine. Thanks!